workflows/VGP-assembly-v2/Pre-Curation_PretextMap/.dockstore.yml

```diff
@@ -0,0 +1,14 @@
+version: 1.2
```

Comment:
The folder name becomes the repository in https://github.com/iwc-workflows.
I would suggest maybe `hi-c-map-generation` ? In any case prefer `-` over `_`.

Reply:
what does pre-curation mean here ? I still think the folder name looks weird. It seems like what this workflow does is generating hi-c (contact?) maps, so why not call it that ?